ansible/playbook.yaml

```diff
@@ -0,0 +1,237 @@
+# Copyright (c) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+- hosts: localhost
+  environment:
+    KUBECONFIG: "/home/<user>/.kube/config"  # Enter your cluster kubeconfig path here
```

Comment:
Could we use the Shell Environment Variable and remove it from the playbook? :-)

Reply:
Yes. Done.